packages/fxa-content-server/app/scripts/models/attached-clients.js

```diff
@@ -12,9 +12,7 @@ import Constants from '../lib/constants';
 import AttachedClient from './attached-client';
 
 var AttachedClients = Backbone.Collection.extend({
-  model(attrs, options) {
-    return new AttachedClient(attrs, options);
-  },
```

Comment:
Oddly, when getting rid of some of the obsolete polyfills via the update to `@babel/core` this broke. Luckily the fix was straightforward.